content/_index.de.md

```diff
@@ -31,21 +31,21 @@ Diese grundlegenden Werte und Prinzipien sind:
 
 * Wir können erfolgreich sein und gleichzeitig vielfältige, erfüllende Leben führen. Unser Erfolg und unser Wert ist nicht davon abhängig, ob wir all unsere Energie in die Entwicklung von Software stecken.
 
-* Wir haben die Verpflichtung unsere - wie auch immer subtil - privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
+* Wir haben die Verpflichtung unsere – wie subtil auch immer – privilegierten Positionen zu nutzen, um das Leben Anderer zu verbessern.
 
 * Wir müssen Raum schaffen für Menschen, die nicht so sind wie wir, damit sie in unseren Bereich einsteigen und darin erfolgreich sind. Das bedeutet nicht nur, diese Menschen in den IT-Bereich einzuladen, sondern auch sicherzustellen, dass sie darin Unterstützung und Ermächtigung finden.
```

Comment:
I would say `... erfolgreich sein können` instead of `... sind` so that the sentence works for the previous `einsteigen`.

Reply:
Hm, I just had another look at the original wording for this part:

```
We must make room for people who are not like us to enter our field and succeed there.
```
I think my intention with the first translation was to emphasize the declarative nature of these actions: In the English wording, people are not only able to succeed in the field, they _actually do_ succeed. Similarly, they are not only able to enter the field, but they _do_ enter.

In German: Ich denke, meine Absicht war es hier, die Tatsaechlichkeit dieser Handlungen zu unterstreichen. Im Englischen ist davon die Rede, dass Menschen nicht nur erfolgreich sein koennen, sondern, dass sie _tatsaechlich_ erfolgreich _sind_. Genauso wird im Englischen auch davon gesprochen, dass Menschen in die IT-Branche nicht nur einsteigen koennen, sondern dass sie faktisch _einsteigen_. 

I do see your point that the sentence reads a bit odd in German though. I'm unsure if I can come up with a better wording than you already did but what do you think about trying to translate the English original even more literally to drive the point home better, e.g.:

```
Wir muessen fuer Menschen, die nicht so sind wie wir, Raum schaffen in unsere Branche einzusteigen und darin erfolgreich zu sein.
```

Not really sure if this suggestion makes it really better, but let me know your thoughts :-)

Reply:
I get what you're saying, and I like the last sentence. However, for Fluidity I think we can still gain a little by moving things around in the sentence: 
```
Wir müssen Raum schaffen für Menschen, die nicht so sind wie wir, damit sie in unserer 
Branche einsteigen und erfolgreich sind.
```

Reply:
Umm, that got me almost word-by-word back to the original translation (except for `Branche`)...

Reply:
Super, I'd be  :+1: for your suggestion